components/brave_wallet_ui/components/desktop/popup-modals/edit-visible-assets-modal/index.tsx

```diff
@@ -230,6 +258,18 @@ const EditVisibleAssetsModal = ({ onClose }: Props) => {
     })
   }, [filteredOutRemovedTokens, searchValue])
 
+  const sortedList = React.useMemo(() => {
```

Comment:
Not really "sorting" in this memo, we are perfoming additional filtering

```suggestion
  const myAssetsOrAvailableAssets = React.useMemo(() => {
```

Reply:
Changed!